SkeletalRepresentationInitializer/Logic/vtkSlicerSkeletalRepresentationInitializerLogic.cxx

```diff
@@ -411,20 +387,19 @@ int vtkSlicerSkeletalRepresentationInitializerLogic::FlowSurfaceMesh(const std::
         points->Modified();
         // TODO: move to the proper directory
         // save the result for the purpose of backward flow
-        char fileName[MAX_FILE_NAME];
-        sprintf(fileName, "%s/forward_output#%04d.vtk", forwardFolder, iter+1);
-
+        std::string fileName;
```

Comment:
If fileName is not modified elsewhere, is preferred to be const correct.

This applies to other strings as well. It helps the compiler to optimize, but also, and specially, the reader of the code.
```cpp
const std::string fileName(forwardFolder + "/" + std::to_string(iter+1) + ".vtk")
```

Reply:
Also, after reading this, the file will start with a number, not familiar if this is cross-platform, will windows like a digit-starting filename? You could add one letter prefix if not.